src/osp/Active/SysNewton.cpp

```diff
@@ -289,6 +290,8 @@ void SysNewton::create_body(ActiveScene& rScene, ActiveEnt entity,
                                 entBody.m_intertia.x(),
                                 entBody.m_intertia.x(),
                                 entBody.m_intertia.z());
+        NewtonBodySetCentreOfMass(entBody.m_body, centerOfMass.xyz().data());
+        entBody.m_centerOfMassOffset = centerOfMass.xyz();
```

Comment:
I just realized, I'm not sure where `entBody.m_mass` currently comes from. Is it set anywhere? Otherwise I'll set it with my own calculated value.

Reply:
It doesn't show up here because it's not directly adjacent to my code changes, but it's 1 line above this window. I suspect that it's never changed from its default initialization of `1.0f` and I should change it.